config/config.exs

```diff
@@ -12,6 +12,7 @@ config :safira,
   company_code: System.get_env("COMPANY_CODE"),
   speaker_code: System.get_env("SPEAKER_CODE"),
   staff_code: System.get_env("STAFF_CODE"),
+  from_email: System.get_env("FROM_EMAIL") || "test@seium.com",
   roulette_cost: String.to_integer(System.get_env("ROULETTE_COST") || "10"),
```

Comment:
Added this fallback email but it's just for dev since it is not a validated email in sendgrid